views/upgrade.hbs

```diff
@@ -0,0 +1,118 @@
+        <span class="upgrade-now-line-1">Upgrade Now</span>
+        <span class="upgrade-now-line-2">$39.99 USD/Year</span>
+      </button>
+    </main>
+    <div class="upgrade-bg-split row-full-width">
+    <section class="upgrade-section flx just-cntr flx-row bg-white upgrade-two-up">
+      <div class="auto upgrade-fade-in">
+        <h2 class="upgrade-subhead upgrade-feature-subhead txt-cntr bolt-stars">
+          Keep all your online accounts, passwords and browsing data <span class="bold">private and safe</span> in one, easy-to-find place.
+        </h2>
+      </div>
+      <div class="upgrade-list-wrapper">
+        <ul class="upgrade-list">
+          {{#each (upgradeList)}}
+            <li class="upgrade-list-item upgrade-fade-in">
+              {{{ this }}}
+            </li>
+          {{/each}}
+        </ul>
+      </div>
+    </section>
+    </div>
+  </div>
+  <div class="flx-col upgrade-bg-light">
+    <div class="row-full-width flx-col">
+      <div class="upgrade-three-up">
+        <section class="upgrade-section">
+          <div class="upgrade-features flx flx-row">
+            {{#each (upgradeFeatureList)}}
+              <div class="upgrade-feature txt-cntr flx flx-col">
+                <h3 class="upgrade-feature-headline {{ this.className }}">{{ this.title }}</h3>
+                <h4 class="upgrade-feature-subtitle">{{ this.subtitle }}</h4>
+                <p class="upgrade-feature-body">{{ this.body }}</p>
+              </div>
+            {{/each}}
+          </div>
+        </section>
+      </div>
+    </div>
+  </div>
+  <div class="bg-blue bg-blue-shapes">
+    <div class="fx-features-wrapper bg-split-reverse row-full-width">
+    <section class="upgrade-section row-full-width fx-features flx flx-col">
+      <div class="flx flx-row jst-cntr">
+        <div class="upgrade-fx-logo"></div>
+        <h2 class="upgrade-subhead fx-features-subhead">And because it’s integrated with Firefox, you also get:</h2>
+      </div>
+      <div class="upgrade-fx-features-list-wrapper flx flx-col upgrade-list-wrapper">
+        <ul class="upgrade-fx-features-list">
+          {{#each (upgradeFirefoxFeatureList)}}
+            <li class="upgrade-list-item upgrade-fx-features-list-item">
+              {{{ this }}}
+            </li>
+          {{/each}}
+        </ul>
+      </div>
+    </section>
+    </div>
+
+    <section class="ready-to-upgrade row-full-width jst-cntr flx-col flx">
+      <h2 class="upgrade-subhead ready-to-upgrade-hl txt-white txt-cntr">Ready to upgrade for complete peace of mind?</h2>
+      <button class="upgrade-now upgrade-btn flx flx-col" data-page-location="upgrade-now-footer-button">
+        <span class="upgrade-now-line-1">Upgrade Now</span>
+        <span class="upgrade-now-line-2">$39.99 USD/Year</span>
+      </button>
+    </section>
+  </div>
+
+{{else}}
+  <h1>Please log in to your account to view this page</h1>
+  <button>Sign In</button>
```

Comment:
This should be an active sign-in button?

Reply:
This is actually just going away since we're redirecting anyone who isn't signed in back to the homepage.